scripts/github-last-contribution.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+import argparse
+import urllib.request
+from bs4 import BeautifulSoup
+'''
+For using this script call the get_usernames() function and pass all the
+usernames in a space separated string.
+'''
+
+
+def last_contribution(user_name):
+    profile_url = "https://github.com/{}/zulip".format(user_name)
+    try:
+        profile_page = BeautifulSoup(
+            urllib.request.urlopen(profile_url), "lxml")
+    #404 page not reachable or page doesn't exist.
+    except urllib.error.URLError:
+        return urllib.error.URLError
```

Comment:
Apologies for bothering you so much. This is my first time using except and raise.